config/initializers/assets.rb

```diff
@@ -12,3 +12,4 @@
 # application.js, application.css, and all non-JS/CSS in the app/assets
 # folder are already added.
 # Rails.application.config.assets.precompile += %w( admin.js admin.css )
+Rails.application.config.assets.precompile += %w( errors.css )
```

Comment:
Is this still needed?

Reply:
When running `rails server` it complained that this configuration was necessary.